.github/workflows/ci.yml

```
@@ -0,0 +1,48 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  check:
+    needs: []
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Set up GitHub Actions"
+      uses: actions/checkout@v2
+    - name: "Set up Python 3.6"
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: "Install Python dependencies"
+      run: |
+        pip install ".[dev]"
+    - name: "Check code format"
+      run: |
+        make check
+
+  test:
+    needs: [check]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

```suggestion
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest]
        python-version: [3.6, 3.7, 3.8]
```

Maybe as the point of this to be deployed on HEPData it really only matters if this runs on Linux, but I like to test MacOS as well usually for the actual pytest suite.

---

I am fine with adding macOS.